ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -71,10 +72,26 @@ private void TransferBackgroundBrush(BorderedCanvas view)
             }
         }
 
+#if WINDOWS_UWP
+        [ThreadStatic]
+#endif
+        private static Brush s_defaultBorderBrush;
+
         /// <summary>
         /// Default brush for the view borders.
         /// </summary>
-        protected readonly Brush _defaultBorderBrush = new SolidColorBrush(Colors.Black);
+        protected Brush DefaultBorderBrush
+        {
+            get
+            {
+                if (s_defaultBorderBrush == null)
+                {
+                    s_defaultBorderBrush = new SolidColorBrush(Colors.Black);
```

Comment:
It will be better perf if instead of creating the black brush it is defined as a resource and pulled from resources

Reply:
This is instantiated just once. I think we don't really have resources for the library itself.

---
In reply to: [166079703](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166079703) [](ancestors = 166079703)

Reply:
@gmangov - if you have a proposal for how to add resources for core components, it could be very useful! There are other components (like TextInput) that have a ton of default resources defined in this way.

---
In reply to: [166141564](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166141564) [](ancestors = 166141564,166079703)